src/main/java/pivotal/ui/pages/ProfilePage.java

```diff
@@ -0,0 +1,149 @@
+package pivotal.ui.pages;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Set;
+import java.util.function.Supplier;
+
+import core.utils.WebElementInteractor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import pivotal.entities.User;
+
+import static pivotal.constants.UserIdentifiers.NAME;
+import static pivotal.constants.UserIdentifiers.USER_NAME;
+import static pivotal.constants.UserIdentifiers.INITIALS;
+
+public class ProfilePage extends BasePage {
+    @FindBy(id = "person_username")
+    private WebElement usernameTextBox;
+
+    @FindBy(id = "person_name")
+    private WebElement nameTextBox;
+
+    @FindBy(id = "person_initials")
+    private WebElement initialsTextBox;
+
+    @FindBy(xpath = "//div[contains(text(),'UpdatedUsername')]")
+    private WebElement usernameField;
+
+    @FindBy(xpath = "//ul[@class='rows read']//li//div[contains(text(),'Updated name')]")
```

Comment:
Please review all the locators and selectors in your pages, many of them can be improved